docs/usage/providers.md

```
@@ -312,6 +312,165 @@
+      "Resource": [
+        "arn:aws:bedrock:*::foundation-model/*"
+      ]
+    }
+  ]
+}
+```
+
+#### **Setup Examples**
+
+**EC2 Instance Setup:**
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)